multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -23,6 +23,9 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
```

Comment:
I think individually overriding environment variables is less fragile than removing them by overriding ExecStart.

In this case, I think that would be:

```
Environment=DOCKER_OPT_IPMASQ=""
Environment=DOCKER_OPT_BIP=""
```